
uint64 timestamp
AuroraData[] data

